internal/ui/table.go

```diff
@@ -326,6 +337,9 @@ func (v *Table) buildRow(row int, data resource.TableData, sk string, pads MaxyP
 			c.SetExpansion(1)
 			c.SetAlign(align)
 			c.SetTextColor(f(data.Namespace, data.Rows[sk]))
+			if m {
+				c.SetBackgroundColor(tcell.ColorDarkGray) //FIXME
```

Comment:
@derailed could you please suggest a better approach to define this highlight color? I'm thinking to add a new attribute to "skin", but this could cause some incompatibility with custom user skins.

Reply:
@bkmeneguello  Thank you for this PR! I am not sure about this feature. I somewhat see the motivation if pods come and go as the results of jobs running and being completed and hence would alter the selection. Perhaps a better approach here would be to track the actual selection and not the selection index in the table. Which I think this what the mark would do for you. Another approach would be to use the filter or a managing resource ie look at pods by deployment or other to only select the resources you care to manipulate by eliminating other resources noise? The deal here is all the commands would now have to deal with multiple selected items potentially which is currently not part of the design. I am not saying this feature is not worth investigating but as a cluster admin I don't find myself having to bulk manipulate resources very often. It might be good for you to detail the use cases here so we can further weight out this effort. ie what would describe or logs on multiple selected resources would actually look like?

Reply:
@derailed I hope I can to demonstrate the usefulness of this feature witch I'm currently using daily basis.
1. I'm not tracking indices, but resource keys, so if the table is sorted or filtered the selected items keep the same (marks is a string slice).
2. My use case is a Jenkins Kubernetes cluster, so pods are created all the time and are short-lived, and sometimes they fail by some reason, so I need to kill them and Jenkins will create another one. But eventually between the time I focus a pod and hit ctrl+k the screen is updated and I kill the wrong pod. Another cause is when I have a large pod failure and need to kill dozens of pods (suffering with the previous problem).
3. Not all command should deal with multiple selection. To this moment I only identified delete/kill, but some more could be added, like taint nodes. To prevent confusing users I tried to hide commands which don't support multiple selection when something is marked, but the actions aren't refreshed (I think is a bug) and didn't investigated further. I think it could be one more option of KeyAction and filter actions automatically.

I hope I made my point clear. This tool is awesome and this feature is really a must have for me. I don't know if you are a Htop user, but it's the same processes used there to mark processes to invoke "kill"

Reply:
@bkmeneguello Thank you so much for the clarification! I get it now and agree with you on the usefulness. I do think the logic your are using for selecting resources is the correct approach and we should generalize it as the correct way to select. I'll also take a closer look and see what's up with the refresh and will assist to push this thru. As for the mark color I think we can introduce a new skin for it as K9s is still very much pre-release. I very much appreciate your kind words, support and this effort to improve K9s for all of us! 

Thank you Bruno!!

Reply:
@derailed, all basic functionality is working. It's ready for review.